web/parts/Dockerfile-node7

```diff
@@ -0,0 +1,11 @@
+arg node_release=node_7.x
```

Comment:
it's the same as node6, maybe just have the `node_release` as a separate file and move out the rest to something like `web/parts/Dockefile-node`